### PR TITLE
clean up rspec_json_formatter

### PR DESCRIPTION
**Pull request by vjeffrey**
hack-days work to clean up rspec_json_formatter a bit.  removed some unnecessary instance variables that were floating around and commented methods

### CODE REVIEW

**Comment by vjeffrey** [comment]
the failure there is a formatting thing that was merged in rspec_json_formatter yesterday

File: lib/inspec/rspec_json_formatter.rb [hunk 1]
```diff
@@ -330,12 +349,45 @@ def status_type(data, control)
...
+
+      print_results(fails + skips + passes)
+    end
+  end
+
+  ############# Current control methods #############
+
+  # Takes current_control (called from flush_current_control) and returns
+  # fails, skips, passes.
+  def current_control_infos(current_control)
```
Comment (chris-rock) [inline lib/inspec/rspec_json_formatter.rb:385]: 👍

File: lib/inspec/rspec_json_formatter.rb [hunk 2]
```diff
@@ -345,9 +397,11 @@ def current_control_infos
     [fails, skips, passes, STATUS_TYPES.key(summary_status)]
   end
 
-  def current_control_title
-    title = @current_control[:title]
-    res = @current_control[:results]
+  # Determine title for control given current_control.
+  # Called from current_control_summary.
+  def current_control_title(current_control)
```
Comment (chris-rock) [inline lib/inspec/rspec_json_formatter.rb:402]: 👍

File: lib/inspec/rspec_json_formatter.rb [hunk 3]
```diff
@@ -280,26 +280,20 @@ class InspecRspecCli < InspecRspecJson # rubocop:disable Metrics/ClassLength
   MULTI_TEST_CONTROL_SUMMARY_MAX_LEN = 60
 
   def initialize(*args)
-    @colors = COLORS
-    @indicators = INDICATORS
-
-    @format = '%color%indicator%id%summary'
```
Comment (chris-rock) [inline lib/inspec/rspec_json_formatter.rb:286]: 👍

**Review by chris-rock — APPROVED** [review]

**Comment by chris-rock** [comment]
Thank you @vjeffrey

**Pull request merged**